Formula/vsftpd.rb

```diff
@@ -44,11 +44,45 @@ def caveats
       s += <<-EOS.undent
         vsftpd was compiled with SSL support. To use it you must generate a SSL
         certificate and set 'enable_ssl=YES' in your config file.
+
       EOS
     end
 
+    s += <<-EOS.undent
+      To use chroot, vsftpd requires root privileges, so you will need to run
+      `sudo vsftpd`.
+      You should be certain that you trust any software you grant root privileges.
+
+      The vsftpd.conf file must be owned by root in order to work correctly:
```

Comment:
Can you elaborate on "work correctly"?  Would be good if there was some way of doing this on first startup.